queries/inko/textobjects.scm

```diff
@@ -0,0 +1,278 @@
+; Classes
+(class) @class.outer
+
+(class
+  body: (class_body
+    .
+    "{"
+    .
+    (_) @_start
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start)))
+
+; Traits
+(trait) @class.outer
+
+(trait
+  body: (trait_body
+    .
+    "{"
+    .
+    (_) @_start
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start)))
+
+; Implementations
+(implement_trait) @class.outer
+
+(implement_trait
+  body: (implement_trait_body
+    .
+    "{"
+    .
+    (_) @_start
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start)))
+
+(reopen_class) @class.outer
+
+(reopen_class
+  body: (reopen_class_body
+    .
+    "{"
+    .
+    (_) @_start
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "class.inner" @_start)))
+
+; Methods and closures
+(method) @function.outer
+
+(method
+  body: (block
+    .
+    "{"
+    .
+    (_) @_start
+    (_)? @_end
+    .
+    "}"
+    (#make-range! "function.inner" @_start)))
```

Comment:
Looks like this directive and many (all?) others are missing the second parameter: should be (for this one)
```query
(#make-range! "function.inner" @_start @_end)
```

Reply:
@ribru17 I removed this in response to your request in https://github.com/nvim-treesitter/nvim-treesitter-textobjects/pull/610#discussion_r1595773398. Did you mean to only remove `@_end` from certain nodes (and if so, which ones)?

Reply:
Sorry, I should have been clearer. Basically, the `@_end` should be removed in `(_) @_start @_end`, but not in the `#make-range!` predicate. That way the make range still uses the end capture, but makes sure it always uses the last end capture (not the one that was also applied to the start capture)

Reply:
@ribru17 Ah gotcha, this should be fixed now :smiley:

Reply:
Thank you!